en/_posts/2016-04-20-marathon-health-checks.md

```diff
@@ -6,23 +6,23 @@ author: devatoria
 lang: en
 ---
 
-Health checks are useful to ensure that your container is running; and better, is running as expected. You can add multiple health checks per application. A health check can be of two types:
+Health checks are useful to ensure that your containers are running; and even better, are running as expected. You can add multiple health checks per application. A health check can be of two types:
 
 - HTTP
 - TCP
 
-The only difference is that the HTTP check is checking a path (using http://localhost:port/path), and is able to ignore some HTTP status code (informational ones, from 100 to 199).
+The only difference is that the HTTP check is given a path (using http://localhost:port/path), and is able to ignore some HTTP status codes (informational ones, from 100 to 199).
```

Comment:
HTTP checks allow for better, higher level checks on a specific URL like ``http://localhost:port/my/health/path``. It automatically ignores informational status codes in the range 100 to 199.